lib/ansible/modules/cloud/azure/azure_rm_networkinterface.py

```diff
@@ -158,6 +158,12 @@
                 type: bool
                 default: 'no'
         version_added: 2.5
+    enable_accelerated_networking:
+        description:
+            - Specifies whether the network interface should be created with the accelerated networking feature or not
+        type: bool
```

Comment:
Could add an alias to `accelerated_networking` and a comment to check whether the `vm_size` supports accelerated networking.